### PR TITLE
Shubham's Code Cleanup 2

### PR DESCRIPTION
**Pull request by shubham3121**
## Description
Please include a summary of the change, the motivation, and any additional context that will help others understand your PR. If it closes one or more open issues, [please tag them as described here](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

## Affected Dependencies
List any dependencies that are required for this change.

## How has this been tested?
- Describe the tests that you ran to verify your changes.
- Provide instructions so we can reproduce.
- List any relevant details for your test configuration.

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [x] My changes are covered by tests

### CODE REVIEW

File: packages/syft/src/syft/core/node/common/client_manager/dataset_api.py [hunk 1]
```diff
@@ -140,9 +139,9 @@ def all(self) -> List[Any]:
         new_all = list()
         for dataset in result:
             new_dataset = {}
-            for k, v_blob in dataset.items():
+            for k, value in dataset.items():
                 if k not in ["str_metadata", "blob_metadata", "manifest"]:
-                    new_dataset[k] = deserialize(v_blob, from_bytes=True)
```
Comment (IonesioJunior) [inline packages/syft/src/syft/core/node/common/client_manager/dataset_api.py:145]: Does this change preserve the old behavior?

Comment (madhavajay) [reply packages/syft/src/syft/core/node/common/client_manager/dataset_api.py:145]: I agree, perhaps we need a test for the dataset api?

Comment (shubham3121) [reply packages/syft/src/syft/core/node/common/client_manager/dataset_api.py:145]: I will add a test to verify this.

**Comment by madhavajay** [comment]
@shubham3121 i think this just needs a check to make sure the nested metadata deserialization isnt broken.

**Comment by shubham3121** [comment]
Closing this PR, since the protobuf will be removed in 0.8

**Pull request closed without merging**